velox/exec/ExchangeClient.h

```diff
@@ -116,7 +116,7 @@ class ExchangeClient {
   std::shared_ptr<ExchangeQueue> queue_;
   std::unordered_set<std::string> taskIds_;
   std::vector<std::shared_ptr<ExchangeSource>> sources_;
-  bool closed_{false};
+  std::atomic_bool closed_{false};
```

Comment:
I checked that all current accesses to `closed_` are all under the protection of `queue_->mutex()`, so there is no need to modify it to `std::atomic_bool`, right?

Reply:
@lingbin Yes, I think this is correct.